modules/cluster-status-section.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * operating/use-system-health-dashboard.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="cluster-status-section_{context}"]
+= Cluster status section
+
+The *Cluster status* section organizes the information into the following groups:
```

Comment:
Are these items in UI order? if there is no order to lean on, suggest alphabetizing

Reply:
yes in UI order